src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -310,14 +310,16 @@ Unless your account belongs to an administrator group, you can only see the logs
 
 - `Job standard output/error logs`: +
 To view the standard output or error logs associated with a job, select a job from the job list and then on the _Output_ tab in the bottom right panel. +
-To view logs of a currently running job, you need to click on the _Streaming_ checkbox, the logs panel will be updated as soon as new log lines will be printed by this job.  +
-If the job is terminated, you can click on the _Fetch output_ button to view the final output of the job. +
+Click on _Streaming Output_ checkbox to auto-fetch logs for running tasks of the entire Job. The logs panel will be updated as soon as new log lines will be printed by this job. +
+You cannot select a specific Task in the streaming mode. If you activate streaming while some Tasks are already finished, you will get the logs of those Tasks as well. +
+Click on _Finished Tasks Output_ button to retrieve logs for already finished tasks. For all the finished Tasks within the Job, or for the selected Task. +
```

Comment:
I think it will be simpler to understand if you say 
"Click on _Finished Tasks Output_ button to retrieve all finished tasks or selected task logs."

Reply:
Text comes from Densi